full-customer-journey/src/app/signIn/page.tsx

```diff
@@ -38,7 +38,7 @@ const page = () => {
         </nav> */}
       </header>
       <section className="flex justify-center gap-x-8 items-center w-full">
-        <div className="w-[610px] flex flex-col items-center  border-2 border-red-300">
+        <div className="w-[610px] flex flex-col items-center ">
           <form className="max-w-[372px]">
             <h2 className="font-semibold  text-4xl text-[#101828] leading-[44px] -tracking-[0.02em] mb-2">
```

Comment:
Make css variables for the colors and use them in your tailwind config. You should know what is your primary color what is secondary color and their variants.